src/Request/AdminFetcher.php

```diff
@@ -34,7 +34,7 @@ public function get(Request $request): AdminInterface
         $adminCode = $request->get('_sonata_admin');
 
         if (!\is_string($adminCode)) {
-            $route = $request->get('_route');
+            $route = $request->get('_route', '');
```

Comment:
```suggestion
            $route = $request->attributes->get('_route', '');
```

Reply:
Isn't this a BC break ? If someone provide a `_route` as a query param or so.
I thought we could only change those code on the next major to solve https://github.com/sonata-project/SonataAdminBundle/issues/7365